src/routes/code-of-conduct/+page.svelte

```diff
@@ -1,14 +1,17 @@
 <script>
```

Comment:
Code of Conduct mending di ilangin dulu, kita belum punya soalnya. Tolong sekalian dihapus dari footer ya